Hashes

HansKiller-TH opened the pull request:
beide Hash Klassen. HashBase hab ich nicht verwendet. Ich sehe da keine sinnvolle Überschneidung außer dem Interface

HansKiller-TH committed:
Hashes

JoJo1541 commented:
Hatte er nicht gesagt, dass wir die Hash-Funktion in der HashBase implementieren sollen?

JoJo1541 commented:
Nach dem Gespräch heute sollte die Hashfunktion in eine eigene Methode verschoben werden.

maxteahan committed:
Merge branch 'main' into HashHans

maxteahan commented:
Habe mal meine SepChain reingemergt. Ist nichts wildes. Lediglich ein Zugriffsmodifikator war anders.

JoJo1541 commented:
In die Branch hier oder nach main? Dann kann man den Request hier zumachen.

maxteahan committed:
Anpassung Print Funktion und Vererbung, Einbinden HashFunc

1. Fix Vererbung. Nun passt Vererbung wieder.

2. HashTabKlassen erben wieder von HashBase.

3. HashFunc ist wie gefordert implementiert. Liegt aktuell in HashBase.

maxteahan commented:
HashFunktion ist jetzt in einer Methode und neu angepasst.

maxteahan committed:
Merge branch 'main' into HashHans
